### PR TITLE
Sprint june 5

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | staging  |
| zboxcli  | sprint-june-5  |
| zwalletcli  | sprint-june-5  |
| 0chain  | sprint-june-5  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | staging  |
| zboxcli  | sprint-june-5  |
| zwalletcli  | sprint-june-5  |
| 0chain  | sprint-june-5  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

File: tests/cli_tests/zboxcli_file_download_test.go [hunk 1]
```diff
@@ -1241,6 +1241,18 @@ func TestDownload(testSetup *testing.T) {
 		filesize := int64(256)
 		remotepath := "/"
 
+		output, err := updateStorageSCConfig(t, scOwnerWallet, map[string]string{
```
Comment (Jayashsatolia403) [inline tests/cli_tests/zboxcli_file_download_test.go:1244]: This test is not sequential.

**Review by Jayashsatolia403 — CHANGES_REQUESTED** [review]

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | readPool-unlock  |
| zboxcli  | sprint-june-5  |
| zwalletcli  | sprint-june-5  |
| 0chain  | sprint-june-5  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

File: tests/cli_tests/zboxcli_file_download_test.go (hunk 1 above)
Comment (Kishan-Dhakan) [reply tests/cli_tests/zboxcli_file_download_test.go:1244]: It's WIP

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-july-1  |
| gosdk  | staging  |
| zboxcli  | sprint-july-1  |
| zwalletcli  | sprint-july-1  |
| 0chain  | show-err-msg  |
| blobber  | sprint-july-1  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-july-1  |
| gosdk  | staging  |
| zboxcli  | fix-repair-complete  |
| zwalletcli  | sprint-july-1  |
| 0chain  | sprint-july-1  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-july-1  |
| gosdk  | readPool-unlock  |
| zboxcli  | sprint-july-1  |
| zwalletcli  | sprint-july-1  |
| 0chain  | sprint-july-1  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-june-5  |
| gosdk  | staging  |
| zboxcli  | sprint-june-5  |
| zwalletcli  | sprint-june-5  |
| 0chain  | sprint-june-5  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-july-1  |
| gosdk  | sprint-july-1  |
| zboxcli  | sprint-july-1  |
| zwalletcli  | sprint-july-1  |
| 0chain  | show-err-msg  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Pull request merged**